.thoth.yaml

```diff
@@ -7,7 +7,7 @@ runtime_environments:
     operating_system:
       name: rhel
```

Comment:
@fridex @harshad16 Should I change it to `ubi`?

Reply:
ubi/rhel are actually synonyms on the recommender side so no need for a change

Reply:
good, thanks